lineapy/plugins/pipeline_writers.py

```diff
@@ -140,9 +141,9 @@ def docker_template_name(self) -> str:
         return "dockerfile.jinja"
 
     def _write_dag(self) -> None:
-        dag_flavor = self.dag_config.get(
-            "dag_flavor", AirflowDagFlavor.PythonOperatorPerSession
-        )
+        dag_flavor = AirflowDagFlavor[
+            self.dag_config.get("dag_flavor", "PythonOperatorPerSession")
```

Comment:
why did we go to string instead of the prev enum?

Reply:
This is more UI reason. Just like when we specify framework for to_pipeline. It's easier for user to specify a string value of PythonOperatorPerSession than find out the class object.

Reply:
Is strenums in 3.7 standard library?